contrib/freebsd/Makefile

```diff
@@ -0,0 +1,53 @@
+# Created by: Norbert Kamiński <norbert.kaminski@3mdeb.com>
+# $FreeBSD$
+
+PORTNAME=	fwupd
+DISTVERSION=	1.5.9
```

Comment:
Don't hardcode this to anything upstream. Leave it as a value that is filled in with sed or awk.

Reply:
Values set during the build.